cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraValueReader.scala

```diff
@@ -43,15 +43,17 @@ class CassandraValueReader(
     val writerSchema = attributeStore.readSchema(layerId)
     val codec = KeyValueRecordCodec[K, V]
 
-    def read(key: K): V = instance.withSession { session =>
-      val statement = session.prepare(
+    private lazy val statement = instance.withSession{ session =>
```

Comment:
Do you think it should be private?

Reply:
I debated... I saw that none of the other instance fields were, but if someone accesses this `statement` from the outside and uses a different `Session` than the one originally used to prepare it then that would cause an exception.  I ended up deciding to keep the field hidden to prevent potential misuse. 
If the project standard is to mark instance fields public though I don't have a problem changing it... just wasn't sure what the standard was.